source/optional_features/block_prepaid_cards/index.html.md.erb

```diff
@@ -0,0 +1,15 @@
+---
+title: Block prepaid cards
+weight: 140
+---
+
+# Block prepaid cards
+
+You can block users from paying with prepaid cards. You should only do this if users are making fraudulent payments with prepaid cards.
+
+You cannot block most Mastercard prepaid cards, because we usually cannot tell if Mastercard cards are prepaid or not.
+
+[Contact us](/support_contact_and_more_information/#contact-us) and we’ll:
+
+- tell you how to get permission from Visa first - if you want to block Visa cards
+- block prepaid cards on your account
```

Comment:
@miriamalice Following 2i and our discussion, I've simplified the above section for now. We can add more detailed steps in another iteration when we've fleshed them out. Let me know if it looks ok!